functions/src/helpers.ts

```diff
@@ -16,23 +16,63 @@ export const createMP3DBRef = (filePath, _module) => {
     const mp3 = parts[2];
     let file_name = mp3.slice(0, -4);
 
-    return `/${_module}/files/${list}/${file_name}`;
+    return `/files/${list}/${file_name}`;
 }
 
 export const extractListFromFilename = (fileName) => {
     return fileName.match(/^[^-]*[^ -]/g)[0];
 }
 
 
-export const storeFileNameToDB = async (filePath, db, _module) => {
+export const storeFileToDB = async (filePath, db, _module) => {
     if(checkValidMP3(filePath)) {
         let ref = db.ref(createMP3DBRef(filePath, _module));
 
         //checking if the file already exists in the RT db
         let snapshot = await ref.child("status").once('value')
 
         if(!snapshot.exists()) {
```

Comment:
Here and above, there are `if`s that span all the function body. I don't like such constructs, they are hardly readable.

Rather negate the condition and return from the function.